core/iwasm/aot/aot_loader.c

```diff
@@ -2076,6 +2063,12 @@ load_relocation_section(const uint8 *buf, const uint8 *buf_end,
         goto fail;
     }
 
+    symbols = loader_malloc(sizeof(*symbols) * symbol_count, error_buf,
```

Comment:
Had better change to `sizeof(*symbols) * (uint64)symbol_count` to avoid integer overflow: `sizeof(*symbols) * symbol_count` may lead to integer overflow firstly in 32-bit system before it is passed to loader_malloc, if symbol_count is large enough.

Reply:
done